### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by oscarnava**
First version of the heat map project.

### CODE REVIEW

**Comment by mcrd25** [comment]
# Currently Under Review

File: index.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Untitled Document</title>
```
Comment (mcrd25) [inline index.html]: Please put a more descriptive Title 😄

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Good attempt at heatmap! 👍 

Hi @oscarnava and @nicolasmarino99 👋,

This was a really great attempt! I know this project could be confusing to some folks. 
Before I get into the more opinionated part of my review (suggestions), first let's take a look at the Minimum requirements.

# Minimum Requirements

#### Navigation bar
- It should use the `<nav>` HTML tag
- it should have seven separate containers (`<div>` is recommended)

#### articles
- Each article should have a title and paragraph with placeholder text
- Articles with images should have an image in the top left corner with the title of an article under it

#### README
- *You've done a fair job here but try to remove the Microverse project heading and put a better description, avoid mentions of Microverse or Odin project and try to specify what the project is about. Also, link your profiles (GitHub, etc.) and put a link to live version. You can use raw.githack.com when in dev. Here is a link I used https://rawcdn.githack.com/oscarnava/smashing-grayscale-heat-map/a2673bc485988e478ac6d09c9fd9099e97e274c4/index.html*

#### Flow
- *You used Git Flow rather than GitHub Flow, which is fine, but please remember to use [GitHub Flow](https://guides.github.com/introduction/flow/) with appropriately named feature branch next HTML/CSS Course. You'd be using Git Flow from the Ruby OOP Project.*
[Here](https://lucamezzalira.com/2014/03/10/git-flow-vs-github-flow/) is reading on Git Flow vs GitHub Flow
 
#### Main content
The main content should be made from separate containers, stacked on top of each other and in the following arrangement:

#### Footer
- The links in the footer should cover the central part of the footer, both horizontally and vertically
- It should have an unorganized list `<ul>`
- The links in the footer should be list items `<li>` positioned using the `float` or `flex` display property 



**Please understand that these requirements would help you become a better developer as it's help your code meet standards and even improve your webpage's SEO when you develop them with the maintained standards. Using `main`, `section` and `article` tags are good practices.**

# Tips
- The original container is the width of 100% maybe you could do the same
- I noticed there is no element within heatmap that is actually really dark or even black, consider an element within the page that immediately catches your attention. For me, it was the "Don’t Miss These Latest Articles" text. 
- Remember there are more than 3 colors in greyscale, maybe try setting a greyscale monochromatic color palette with several shades each with the level of importance (black being most important) and using that as you identify the areas in the webpage that catch your immediate attention, etc.
- In general, your code is neat! keep up great work!
- Impressed by your use of SASS! **REALLY GREAT JOB!**

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  <link rel="stylesheet" href="styles/styles.css">
+</head>
+
+<body>
+
+  <div class="container">
+
+    <div id="row-01" class="full-width">
```
Comment (mcrd25) [inline index.html:14]: This is your nav correct? Try putting it within a `nav` tag and putting some containing tag like `div` to represent the seven links in the original page's navbar with appropriate heatmap color.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+      <div class="heat-100">
+        <p>Fusce in urna sem. Vivamus vehicula dignissim augue et scelerisque.</p>
+      </div>
+    </div>
+
+    <div id="row-02" class="two-columns">
+      <div class="heat-75">
```
Comment (mcrd25) [inline index.html]: Try to include the image, title and article body sections within each article.

Maybe wrap the article in `article` tag

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Comment by Torres-ssf** [comment]
### Great work making the suggested changes 👏
I'm giving this project a 👍. Go on and merge to master. Good luck with the next one.

**Pull request merged**